CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+**Table of Contents**
+- [Getting Started](#getting-started) (1st draft)
+- [Philosophy](#philosophy) (TBD)
+- [Reporting an Issue](#reporting-an-issue) (TBD)
+- [Pull Requests](#pull-requests) (1st draft)
+	-  _Should this be renamed to "fixing an issue"?_
+	-  Link to GitHub guide for PRs
+	-  What issues are good to start with? (Eg. `feature`, `bug`, etc. )
+- [Setup and Run](#setup-and-run) (TBD)
+## Getting Started
+Hi, and welcome to highlight.js. We're a code-highlighting engine etc., but you probably know that. You want to contribute, and this document is the your guide.
+
+There are many ways to contribute to highlight.js. Feel free to work on an open issue. You can also help by writing [a plugin](https://highlightjs.readthedocs.io/en/latest/plugin-api.html) or creating [a theme](https://highlightjs.readthedocs.io/en/latest/style-guide.html). In terms of getting help, GitHub issues are the primary method of communication. We're exploring the idea of creating a Gitter chat room.
```

Comment:
Isn't reporting an issue also contributing?

Perhaps mention creating a 3rd party grammar here (with link to the quick start for that)?

Reply:
This https://highlightjs.readthedocs.io/en/latest/language-guide.html? Or the md document?

Reply:
I was thinking of `extra/3RD_PARTY_QUICK_START.md` which was thrown together to answer "I want to make a 3rd party grammar, what do I do?"... Surely it could be improved also. :)